src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -18,6 +20,15 @@ public function storageData(int $donationFormId): array
             'blockData' => get_post($donationFormId)->post_content,
             'settings' => get_post_meta($donationFormId, 'formBuilderSettings', true),
             'currency' => give_get_currency(),
+            'templates' => array_map(static function ($templateClass) {
+                /** @var FormTemplate $template */
+                $template = give($templateClass);
+                
+                return [
+                    'id' => $template::id(),
+                    'name' => $template::name(),
```

Comment:
Just want to point out this change to the form template contract. I initially  had getters for id and name that were not necessary because we can simply use the static methods directly.

Reply:
Cool. I was looking at that change - sounds good to me!